include/aero/actuator/ActuatorBulk.h

```diff
@@ -67,6 +68,16 @@ struct ActuatorBulk
 
   void stk_search_act_pnts(
     const ActuatorMeta& actMeta, stk::mesh::BulkData& stkBulk);
+
+  //Generic stk search
+  void stk_search(
+    const ActuatorMeta& actMeta, stk::mesh::BulkData& stkBulk, bool onlyFine = false);
+
+  //do I need this to be virtual function so the generic stk_search can call it, even though 
+  //the turbine search only makes sense in the ActuatorBulkFast context as implemented? 
+  virtual void stk_turbine_search(
```

Comment:
No I wouldn't make this a virtual function and I wouldn't define it here. I think it would make more sense to have `stk_search` be virtual and then implement the conditionals and turbine search in the child classes that are actually turbines.